ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -157,6 +157,12 @@ do
   yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "$SHARED_DIR/install-config.yaml" "$f"
 done
 
+# workaround for bug https://issues.redhat.com/browse/OCPBUGS-33921
```

Comment:
just remove these lines in a separate commit if you want to keep them in a log temporarily. This way the pj-rehearse plugin won't catch other jobs as changed needless.